nat/service_iptables.go

```diff
@@ -43,6 +45,17 @@ func (service *serviceIPTables) Stop() error {
 }
 
 func (service *serviceIPTables) enableIPForwarding() (err error) {
+	out, err := exec.Command("sysctl", "-n", "net.ipv4.ip_forward").CombinedOutput()
```

Comment:
Would be nice to that `enableIPForwarding` would have 1 responsibility.
Suggest to move this logic to another function e.g. `isIPForwardingEnabled()`